examples/olsconfig.yaml

```diff
@@ -32,6 +32,13 @@ ols_config:
     lib_log_level: warning
   default_provider: my_bam
   default_model: ibm/granite-13b-chat-v2
+# question_filters:
```

Comment:
is the idea to have it commented to disable filtering?
Won't it be better to use some enable/disable option?

Reply:
This is a example file. We dont know what the real users regex and replacement words will be.